src/lib/mdns/Advertiser.h

```diff
@@ -150,7 +150,7 @@ class ServiceAdvertiser
     virtual ~ServiceAdvertiser() {}
 
     /// Starts the advertiser. Items 'Advertised' will become visible.
-    /// May be called before OR after Advertise() calls.
+    /// Must be called before Advertise() calls.
```

Comment:
This conflicts with the implementation in server/Mdns.cpp: there we do a Start followed by Advertise